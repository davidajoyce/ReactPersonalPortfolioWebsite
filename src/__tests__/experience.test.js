import React from 'react'
import experience from '../Components/experience'
import { shallow } from 'enzyme';

describe('experience', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<experience />);
    expect(wrapper).toMatchSnapshot();
    // On the first run of this test, Jest will generate a snapshot file automatically.
  });
});