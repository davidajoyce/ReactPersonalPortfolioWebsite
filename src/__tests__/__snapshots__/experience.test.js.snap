// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`experience renders correctly 1`] = `ShallowWrapper {}`;
